views/frontend/post.php

```diff
@@ -93,7 +59,8 @@
 
                     <div class="col s12">
                         <button type="submit" name="submit" class="btn waves-effect" 
-                                onclick="return window.confirm(`Ce commentaire va être validé par notre équipe avant d'être affiché`)">
+                                onclick="return window.confirm(`Ce commentaire va être validé par notre équipe avant d'être affiché`)"
+                            href="/article/<?= transforme_en_url($post->title) ?>-<?= $post->id ?>">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$post'.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5344015)